t/data_migration_pipeline/main.go

```diff
@@ -79,6 +80,14 @@ func main() {
 	// Convert each line to a data model
 	dataModels := emitResult(s, lines)
 
+	// Turn on dry run monitor
+	log.Println(*dryRun)
+	if *dryRun {
+		log.Println("dry run start")
```

Comment:
Usually in a dry run, we also need to know metrics like, e.g. total count of records, how many extra processing we did (e.g. backfill an empty ID). You can use beam.NewCounter to output and print.

Reply:
Done